server/controllers/userController.js

```diff
@@ -72,7 +72,7 @@ async function getUser(req, res, next) {
 }
 
 function createTokenResponse(user, res) {
-  const payload = { user };
+  const payload = { user: user._id };
```

Comment:
Maybe we can name this userId so it's more clear that we're just passing the id and not the whole user to passport above

Reply:
Done!